lib/lightning/LndClient.ts

```diff
@@ -40,6 +40,8 @@ interface GrpcResponse {
 interface LndClient {
   on(event: 'invoice.paid', listener: (invoice: string) => void): this;
   emit(event: 'invoice.paid', invoice: string): boolean;
+  on(event: 'invoice.failed', listener: (rHash: Buffer) => void): this;
```

Comment:
This should also have the arguments ` invoice: string` like the `emit` of the `invoice.failed` event
```suggestion
  on(event: 'invoice.failed', listener: (invoice: string) => void): this;
```